index.html

```diff
@@ -38,9 +39,137 @@ <h2 class="intro profession">I'am a Software Developer</h2>
             <li><a href="#" class="fa fa-medium"></a></li>
         </ul>
     </div>
+</section>
+
+<section class="item-2">
+    <div class="recentWork">
+        <h1>My Recent Works</h1>
```

Comment:
- [ ] Kindly use a `h2` instead of `h1`. The `h1` tag is just for the title of the website.